monkestation/code/modules/gauntlet/syndie.dm

```diff
@@ -0,0 +1,245 @@
+		passenger = target.pulling
+		holder.passenger = passenger
+
+	target.visible_message("<span class='danger bold'>[target] LEAPS[passenger ? ", bringing [passenger] up with them" : ""]!</span>")
+	target.opacity = FALSE
+	target.mouse_opacity = FALSE
+	if(iscarbon(target))
+		var/mob/living/carbon/C = target
+		C.super_leaping = TRUE
+	if(passenger)
+		passenger.opacity = FALSE
+		passenger.mouse_opacity = FALSE
+		animate(passenger, pixel_y = 128, alpha = 0, time = 4.5, easing = LINEAR_EASING)
+	animate(target, pixel_y = 128, alpha = 0, time = 4.5, easing = LINEAR_EASING)
+	sleep(4.5)
+
+	if(passenger)
+		passenger.forceMove(holder)
+		passenger.reset_perspective(holder)
+		passenger.notransform = FALSE
+	target.forceMove(holder)
+	target.reset_perspective(holder)
+	target.notransform = FALSE //mob is safely inside holder now, no need for protection.
+
+	sleep(7.5 SECONDS)
+
+	if(target.loc != holder && (passenger && passenger.loc != holder)) //mob warped out of the warp
+		qdel(holder)
+		return
+	mobloc = get_turf(target.loc)
+	target.mobility_flags &= ~MOBILITY_MOVE
+	if(passenger)
+		passenger.mobility_flags &= ~MOBILITY_MOVE
+	holder.reappearing = TRUE
+
+	if(passenger)
+		passenger.forceMove(mobloc)
+		passenger.Paralyze(75)
+	target.visible_message("<span class='danger bold'>[target] slams down from above[passenger ? ", slamming [passenger] down to the floor" : ""]!</span>")
+	playsound(target, 'sound/effects/bang.ogg', 50, 1)
+	explosion(mobloc, 0, 0, 2, 3)
+	target.forceMove(mobloc)
+	target.setDir(holder.dir)
+	animate(target, pixel_y = 0, alpha = 255, time = 4.5, easing = LINEAR_EASING)
+	if(passenger)
+		passenger.setDir(holder.dir)
+		animate(passenger, pixel_y = 0, alpha = 255, time = 4.5, easing = LINEAR_EASING)
+	sleep(4.5)
+	target.opacity = initial(target.opacity)
+	target.mouse_opacity = initial(target.mouse_opacity)
+	if(iscarbon(target))
+		var/mob/living/carbon/C = target
+		C.super_leaping = FALSE
+	if(passenger)
+		passenger.opacity = initial(passenger.opacity)
+		passenger.mouse_opacity = initial(passenger.mouse_opacity)
+	for(var/mob/living/L in mobloc)
+		if(L.stat == DEAD || L.InCritical())
+			L.visible_message("<span class='danger bold'>[L] is pancaked by [target]'s slam!</span>")
+			new /obj/item/reagent_containers/food/snacks/pancakes(mobloc)
+			L.gib()
+	qdel(holder)
+	if(!QDELETED(target))
+		if(mobloc.density)
+			for(var/direction in GLOB.alldirs)
+				var/turf/T = get_step(mobloc, direction)
+				if(T)
+					if(target.Move(T))
+						break
+		target.mobility_flags |= MOBILITY_MOVE
+	if(!QDELETED(passenger))
+		passenger.mobility_flags |= MOBILITY_MOVE
+
+/////////////////////////////////////////////
+/////////////// SNOWFLAKE CODE //////////////
+/////////////////////////////////////////////
+
+/mob/living/carbon
+	var/mario_star = FALSE
+	var/super_mario_star = FALSE
+	var/super_leaping = FALSE
+
+/mob/living/carbon/ex_act(severity, target, origin)
+	if(super_leaping)
```

Comment:
use `TRAIT_BOMBIMMUNE` - it does exactly this

Reply:
Neat